docs/linking/unlinking.puml

```diff
@@ -32,18 +32,18 @@ deactivate PISP
 ' TODO: Do we include the ALS lookup here to figure out which Auth service is
 ' responsible for the given Consent?
 
-Switch -> Auth ++:  ""DELETE /consents/123""
+Switch -> Auth ++:  ""POST /consents/123/revoke""
 Auth --> Switch: ""202 Accepted""
 deactivate Switch
 
 Auth -> Auth: Delete the record of the consent.
 
-Auth -> Switch ++: ""DELETE /consents/123""
+Auth -> Switch ++: ""PATCH /consents/123 \n{ status: string,\n  revokedAt: Date}""
```

Comment:
Just to confirm -- this won't be a true JS Date() type right? We need to make it a string since JSON doesn't support dates...

Reply:
Yes sorry -  @lewisdaly  and I are currently discussing the format of the string ( ISO/UTC/DateString/etc.)
Not sure what the Mojaloop standard is - any thoughts @eoln  @jgeewax ?

Reply:
I would suggest UTC format